### ISSUE TITLE
Source code JARs are incomplete

### ISSUE THREAD

**Issue by wborn** [ops4j/org.ops4j.pax.logging#531]
It is annoying to debug Pax Logging code because the third party code is missing from the source code JARs.

E.g. if you debug `pax-logging-log4j2` the code of `log4j-core` is missing from the source code JAR.
So while debugging you first have to figure out what dependency version is used and then continuously switch between 
the `pax-logging-log4j2` and `log4j-core` source code JARs while stepping through the code.

There are probably similar issues while debugging `pax-logging-logback`.

**Comment by grgrzybek** [comment]
I agree - I've been doing it for almost 10 years ;) With proper version of original Log4j2/Logback on debug path it's easier.
I didn't have time to look at it, but feel free to submit a PR :)

**Comment by wborn** [comment]
10 years is a lot! :upside_down_face: 

I see the POM has [commented parts regarding the source code](https://github.com/ops4j/org.ops4j.pax.logging/blob/dfcd0cde3d96c76d884c503a87d417a34dad1907/pax-logging-log4j2/pom.xml#L88-L148). Maybe you already gave it a try but it didn't work well?

**Comment by grgrzybek** [comment]
I don't exactly remember what happened 4 years ago ;) But it could be a good base for a PR...

**Comment by wborn** [comment]
> But it could be a good base for a PR

I think the XML in the POM got commented when the maven-ant-plugin was upgraded to 3.0.0 @grgrzybek.

https://maven.apache.org/plugins/maven-antrun-plugin/

> ## Major Version Upgrade to version 3.0.0
>
> Please note that the following parameters have been completely removed from the plugin configuration:
>
> * `tasks`: use `target` instead;

With a few tweaks it worked again and I was able to use similar constructs for adding all other missing source code in #533.

**Comment by grgrzybek** [comment]
Thanks for the PR - at first and last glance it's perfect - I just didn't have time to merge it ;)
I'm quite busy recently, so give me some time.
